### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by ardatan**
Mention error extensions handling as we do in Yoga
https://the-guild.dev/graphql/yoga-server/docs/features/error-masking#modifying-http-status-codes-and-headers

Related https://github.com/dotansimha/graphql-yoga/pull/3559#pullrequestreview-2511776345

Mention how to extend OpenAPI document
https://github.com/dotansimha/graphql-yoga/discussions/2015#discussioncomment-5940661

### CODE REVIEW

File: website/src/pages/docs/api/error-handler.mdx [hunk 1]
```diff
@@ -21,3 +21,26 @@ api.use(
   })
 );
 ```
+
```
Comment (Urigo) [inline website/src/pages/docs/api/error-handler.mdx:24]: that's nice
can you also on the top section show an example of the default response response that you'll get if you don't add anything and there is an error?

and also for each example - can you also show the actual full response that would be sent?

**Review by Urigo — CHANGES_REQUESTED** [review]

**Comment by Urigo** [comment]
maybe also add error handling to the example on the repo?

File: website/src/pages/docs/api/error-handler.mdx (hunk 1 above)
Comment (ardatan) [reply website/src/pages/docs/api/error-handler.mdx:24]: How about now?

**Comment by ardatan** [comment]
Done @Urigo 👍

File: website/src/pages/docs/api/error-handler.mdx (hunk 1 above)
Comment (Urigo) [reply website/src/pages/docs/api/error-handler.mdx:24]: nice!
I would also add the full json itself, easier to read

**Comment by Urigo** [comment]
what about partial queries?
like, in case some resolvers are ok and some fire an error?

**Comment by ardatan** [comment]
@Urigo I added an example with more details and showing the partially failed response. Let me know what do you think?

File: website/src/pages/docs/api/error-handler.mdx (hunk 1 above)
Comment (ardatan) [reply website/src/pages/docs/api/error-handler.mdx:27]: I added more details including the expected response fully.

**Comment by ardatan** [comment]
I added a link to SOFA docs @Urigo . What do you think?

**Review by Urigo — APPROVED** [review]
@ardatan this is some high level PR and documentation!
Super impressive!

**Pull request merged**